plio/views.py

```diff
@@ -236,6 +236,29 @@ def download_data(self, request, uuid):
         # schema name to query in
         schema_name = OrganizationTenantMiddleware().get_schema(self.request)
 
+        # if the plio belongs to an org schema
+        is_org_plio = False if (schema_name == "public") else True
+
+        # if the plio belongs to an org, is the user requesting the data
+        # an org admin or not
+        is_user_org_admin = False
+        if is_org_plio:
+            organization_shortcode = (
+                OrganizationTenantMiddleware.get_organization_shortcode(self.request)
+            )
+            if OrganizationUser.objects.filter(
+                organization__shortcode=organization_shortcode,
+                user=self.request.user.id,
+                role=Role.objects.filter(name="org-admin").first().id,
+            ).exists():
+                is_user_org_admin = True
```

Comment:
it's more or less same
```suggestion
            is_user_org_admin = OrganizationUser.objects.filter(
                organization__shortcode=organization_shortcode,
                user=self.request.user.id,
                role=Role.objects.filter(name="org-admin").first().id,
            ).exists()
```